test/TBD/installapi-flag.swift

```diff
@@ -3,7 +3,7 @@
 
 // 1. Emit two TBDs, one with -tbd-is-installapi set, and one without
 
-// RUN: %target-swift-frontend -emit-ir -o /dev/null %s -tbd-is-installapi -emit-tbd -emit-tbd-path %t/flag-provided.tbd
+// RUN: %target-swift-frontend -typecheck -o /dev/null %s -tbd-is-installapi -emit-tbd -emit-tbd-path %t/flag-provided.tbd
 // RUN: %target-swift-frontend -emit-ir -o /dev/null %s -emit-tbd -emit-tbd-path %t/flag-omitted.tbd
```

Comment:
Seems weird that these two aren't symmetrical anymore?

Reply:
I could change the second one, sure. But in -tbd-is-installapi mode, we can’t emit IR anymore.

Reply:
Yeah, I just meant the comment makes it sound like they're supposed to be the same except for this one flag.

Reply:
I'll change the other flag, then. No real reason to keep it `-emit-ir`.